src/components/Contributors/Contributors.stories.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import Container from '@material-ui/core/Container';
+import faker from 'faker';
+import Component from '.';
+
+export default {
+    title: 'Components/Contributors',
+    component: Component,
+};
+
+const makeBaseTeam = () => ({
+    fullName: `${faker.name.firstName()} ${faker.name.lastName()}`,
+    description: faker.lorem.paragraph(),
+    subtitle: faker.random.words(2),
+    startDate: faker.date.recent(),
+    endDate: faker.date.future(),
+});
+
+const makeTeam = (num: number) => {
+    const teamMembers = [];
+    for (let i = 0; i < num; i += 1) {
+        const tempMem = makeBaseTeam();
+        if (i % 2 === 0) {
+            teamMembers.push({
+                ...tempMem,
+                picturePath: faker.image.imageUrl(),
+                references: [
+                    {
+                        icon: <></>,
+                        name: 'LinkedIn',
+                        link: 'https://www.linkedin.com',
+                    },
+                ],
+            });
+        } else {
+            teamMembers.push({
+                ...tempMem,
+                picturePath: faker.image.imageUrl(),
+                references: [
+                    {
+                        icon: <></>,
+                        name: 'Email',
+                        link: 'https://www.gmail.com',
+                    },
+                ],
+            });
+        }
```

Comment:
This could be optimized. It doesn't really matter since it's in a story, but I think it's worthwhile to at least see the optimization for learning purposes. 

```js
const teamMembers = [];
const linkedIn = { icon: <></>, name: 'LinkedIn', link: 'https://www.linkedin.com' };
const email = { icon: <></>, name: 'Email', link: 'https://www.gmail.com' };
for (let i = 0; i < num; i += 1) {
    const tempMem = makeBaseTeam();
    teamMembers.push({
         ...tempMem,
         picturePath: faker.image.imageUrl(),
         // this is just creating a copy of linkedIn or email object each time
         references:  i % 2 === 0 ? [{ ...linkedIn }] : [{ ...email }] 
    });
}
```

~27 to ~12 lines give or take a few depending on formatting + clarity is maintained.

Reply:
Thanks for the suggestion